packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -573,8 +574,7 @@ export class SolidParser {
             }
 
             if (SolidParser._IsGroupElement(line) || SolidParser._IsObjectElement(line)) {
-                lineLines.push([]);
-                currentGroup = lineLines[lineLines.length - 1];
+                lineLines.push(currentGroup);

```

Comment:
you are missing resetting the currentGroup to []

Reply:
Damned... :-( Sorry, I am under such a pressure that I'm starting to loose focus. Of course that doesn't excuse my mistake (although I tried one correct obj and one incorrect and I didn't spot the incorrect rendering in the second one), that's why I slapped myself! :-D

Reply:
nw it is what reviews are for :-)